src/arrays/arraymath.jl

```diff
@@ -59,7 +59,7 @@ function *{B<:OrthonormalBasis}(qm1::QuMatrix{B}, qm2::QuMatrix{B})
                    bases(qm2,2))
 end
 
-*(dm1::DualMatrix, dm2::DualMatrix) = (dm1.qarr*dm2.qarr)'
+*(dm1::DualMatrix, dm2::DualMatrix) = QuArray(coeffs(dm1)*coeffs(dm2),bases(dm1,1), bases(dm2,2))
```

Comment:
I guess this change is the reason why Travis fails. The old version returns a DualMatrix, while this returns a QuArray. I agree, that we should try to avoid accessing fields directly, but the old version is more concise. Maybe we need some function to access the array in a DualMatrix?